proxy/src/main/java/org/apache/rocketmq/proxy/processor/ReceiptHandleProcessor.java

```diff
@@ -175,9 +175,10 @@ protected CompletableFuture<MessageReceiptHandle> startRenewMessage(MessageRecei
                 return CompletableFuture.completedFuture(null);
             }
             if (current - messageReceiptHandle.getConsumeTimestamp() < proxyConfig.getRenewMaxTimeMillis()) {
+                RetryPolicy renewPolicy = messageReceiptHandle.getRenewStrategyPolicy();
                 CompletableFuture<AckResult> future =
                     messagingProcessor.changeInvisibleTime(context, handle, messageReceiptHandle.getMessageId(),
-                        messageReceiptHandle.getGroup(), messageReceiptHandle.getTopic(), proxyConfig.getRenewSliceTimeMillis());
+                        messageReceiptHandle.getGroup(), messageReceiptHandle.getTopic(), renewPolicy.nextDelayDuration(messageReceiptHandle.getRenewRetryTimes()));
```

Comment:
We need another counter to save the renew times
The retry times will be increased when some error occur, and will reset to 0 after success

Reply:
The renew times have no relationship with retry times.